<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>404</width>
    <height>568</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Coordinate conversion</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Resource/Resource.qrc">
    <normaloff>:/icon/App</normaloff>:/icon/App</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_6">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>Source coordinate:</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="4" column="0">
            <widget class="QLineEdit" name="leSrcLat"/>
           </item>
           <item row="2" column="0">
            <widget class="QLineEdit" name="leSrcLong"/>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Latitude:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Longitude:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QComboBox" name="cbSrcCoor"/>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>Destination coordinate:</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="2" column="0">
            <widget class="QLineEdit" name="leDstLong"/>
           </item>
           <item row="4" column="0">
            <widget class="QLineEdit" name="leDstLat"/>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Longitude:</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Latitude:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QComboBox" name="cbDstCoor"/>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pbConversion">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Conversion</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QGroupBox" name="groupBox_3">
      <property name="title">
       <string>Transform File</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Source File:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="leSrcFile"/>
         </item>
         <item>
          <widget class="QPushButton" name="pbBrowsSrcFile">
           <property name="text">
            <string>Browse</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Destination File:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="leDstFile"/>
         </item>
         <item>
          <widget class="QPushButton" name="pbBrowsDstFile">
           <property name="text">
            <string>Browse</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pbConversionFile">
           <property name="text">
            <string>Conversion</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QGroupBox" name="groupBox_4">
      <property name="title">
       <string>Transform files in path</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Source Directory:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="leSrcDir"/>
         </item>
         <item>
          <widget class="QPushButton" name="pbSrcDir">
           <property name="text">
            <string>Browse</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Destination Directory:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="leDstDir"/>
         </item>
         <item>
          <widget class="QPushButton" name="pbDstDir">
           <property name="text">
            <string>Browse</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pbConversionDir">
           <property name="text">
            <string>Conversion</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>404</width>
     <height>28</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHelp">
    <property name="toolTip">
     <string>Help</string>
    </property>
    <property name="statusTip">
     <string>Help</string>
    </property>
    <property name="whatsThis">
     <string>Help</string>
    </property>
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout_A"/>
    <addaction name="actionUpdate_U"/>
   </widget>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="sizeGripEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <action name="actionAbout_A">
   <property name="icon">
    <iconset resource="Resource/Resource.qrc">
     <normaloff>:/icon/App</normaloff>:/icon/App</iconset>
   </property>
   <property name="text">
    <string>About(&amp;A)</string>
   </property>
   <property name="iconText">
    <string>About</string>
   </property>
   <property name="toolTip">
    <string>About</string>
   </property>
   <property name="statusTip">
    <string>About</string>
   </property>
   <property name="whatsThis">
    <string>About</string>
   </property>
  </action>
  <action name="actionUpdate_U">
   <property name="text">
    <string>Update(&amp;U)</string>
   </property>
   <property name="iconText">
    <string>Update</string>
   </property>
   <property name="toolTip">
    <string>Update</string>
   </property>
   <property name="statusTip">
    <string>Update</string>
   </property>
   <property name="whatsThis">
    <string>Update</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>cbSrcCoor</tabstop>
  <tabstop>leSrcLong</tabstop>
  <tabstop>leSrcLat</tabstop>
  <tabstop>cbDstCoor</tabstop>
  <tabstop>leDstLong</tabstop>
  <tabstop>leDstLat</tabstop>
  <tabstop>pbConversion</tabstop>
  <tabstop>leSrcFile</tabstop>
  <tabstop>pbBrowsSrcFile</tabstop>
  <tabstop>leDstFile</tabstop>
  <tabstop>pbBrowsDstFile</tabstop>
  <tabstop>pbConversionFile</tabstop>
 </tabstops>
 <resources>
  <include location="Resource/Resource.qrc"/>
 </resources>
 <connections/>
</ui>
